doc/sphinx/source/inspection/index.rst

```diff
@@ -0,0 +1,30 @@
+.. inspection/index: 
+
+Visualization Tools
+###################
+
+One option for visualizing the graph or node structure of a graph is to compile 
+with the :envvar:`NGRAPH_ENABLE_VISUALIZE_TRACING` flag. This generates a PDF, 
```

Comment:
Not `NGRAPH_ENABLE_VISUALIZE_TRACING`. The flags `NGRAPH_ENABLE_*_TRACING` are for developers only because they generate intermediate graphs of the state between graph passes. These flags do not show the original source graph. We need to add a new flag to visualize the source graph. Twice in the last week I have gotten bug reports because users used these flags. They are evil. (the flags, not the users)
The output is png by default but can be controlled by setting `NGRAPH_VISUALIZE_TREE_OUTPUT_FORMAT`